packages/components/css-modules.d.ts

```diff
@@ -1,4 +1,4 @@
 declare module "*.module.css" {
-  const content: Record<string, string>;
-  export default content;
+  const classes: Record<string, string>;
+  export default classes;
```

Comment:
curious, was this needed to work with the plugin?

Reply:
It wasn't, I just decided to go with something consistent with what was in their docs.